powerhome/playbook
# [PLAY-329]- Upgrade Font Awesome to 6.2.0

#### Screens

![Screen Shot 2022-09-21 at 11 41 09 AM](https://user-images.githubusercontent.com/73710701/191550631-6a5f1424-d6ee-4902-89d9-bd1fec5461ab.png)

#### Breaking Changes

No breaking changes, upgraded font awesome to 6.2.0, it is [backward compatible](https://fontawesome.com/v6/docs/web/setup/upgrade/whats-changed#backward-compatibility)

#### Runway Ticket URL

[Runway Story](https://nitro.powerhrg.com/runway/backlog_items/PLAY-329)

#### How to test this

- Test  kits that use icons to make sure icons are rendering correctly. Examples: Icon kit, Circle Icon kit, Selectable Icon Card kit, etc. 
- Test React as well as Rails version of these kits.

#### Checklist:
